app/styles/notification.css

```diff
@@ -1,8 +1,12 @@
+.notifications-holder {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
 .new-message-box {
-    margin-left: 300px;
-    margin-top: 30px;
-    padding-left: 20px;
-    max-width: 800px;
+    width: 800px;
```

Comment:
Was there any specific scenario/reason due to which you went ahead with`width:800px`

Reply:
After making it into flex, the notifications started taking 100% of width, did not want that. So I hardcoded the existing value of max-width and made it to width. Making this dynamic might not be a good idea in my opinion, because the content is limited, extending the notification holders while the text length remains same, might look a bit off. Let me know what you think :)

Reply:
Are we handling the same in different devices?

Reply:
Would we observe viewing issues in different screens for example `2k screen or a mobile`
Is it visually tested for the same?

Reply:
Could we have a mobile view too please

Reply:
@swarajpure